inventory-management/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "inventory",
+  "version": "1.0.0",
+  "description": "Micro-service for inventory management",
+  "main": "app.js",
+  "config": {
+    "appName": "inventory"
```

Comment:
Also I figure the name of the app should be the same as the name of the container, for simplicity.